documentation/examples/remote_storage/remote_storage_adapter/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/blang/semver"
```

Comment:
Oh! Used by another vendored dep so far, but not by Prometheus itself. Was wondering where we were using it :) Ok then.